src/Momento.Sdk/Config/Configuration.cs

```diff
@@ -34,31 +56,34 @@ public Configuration(ILoggerFactory loggerFactory, IRetryStrategy retryStrategy,
     /// <param name="retryStrategy">Defines a contract for how and when to retry a request</param>
     /// <param name="middlewares">The Middleware interface allows the Configuration to provide a higher-order function that wraps all requests.</param>
     /// <param name="transportStrategy">This is responsible for configuring network tunables.</param>
-    /// <param name="loggerFactory">This is responsible for configuraing logging.</param>
-    public Configuration(ILoggerFactory loggerFactory, IRetryStrategy retryStrategy, IList<IMiddleware> middlewares, ITransportStrategy transportStrategy)
+    /// <param name="loggerFactory">This is responsible for configuring logging.</param>
+    /// <param name="readConcern">The client-wide setting for read-after-write consistency.</param>
+    public Configuration(ILoggerFactory loggerFactory, IRetryStrategy retryStrategy, IList<IMiddleware> middlewares,
+        ITransportStrategy transportStrategy, ReadConcern readConcern)
```

Comment:
You can set a default value to read concern `ReadConcern readConcern = ReadConcern.Balanced` I believe. Then no need for the overload.

Reply:
I was trying to think if having two constructors would be better than a default argument for backwards compatibility if we add more arguments in the future. I'm going to make it nullable anyway and have it not explicitly set by default.

Reply:
We should just go with a default value. Otherwise we would have to document what is going to happen when it is null, which will be we set to the default value.